src/commands.ts

```diff
@@ -148,22 +175,51 @@ const unsubCmd = new Command(
     if (!alias) {
       bot.sendMessage(
         channel,
-        `You need to provide the name of the game you want to unsubscribe from.\n`
-        + `Try ${unsubCmd.getTriggerLabel(channel)}.`,
+        `You need to provide the name of the game you want to unsubscribe from.\n` +
+          `Try ${unsubCmd.getTriggerLabel(channel)}.`,
       );
     }
 
-    for (const game of Game.getGames()) {
-      if (game.hasAlias(alias)) {
-        if (bot.removeSubscriber(channel, game)) {
-          bot.sendMessage(channel, `You are now unsubscribed from the **${game.label}** feed!`);
-        } else {
-          bot.sendMessage(
-            channel,
-            `You have never subscribed to the **${game.label}** feed in the first place!`,
-          );
+    const aliases = alias.split(', ');
+
+    // The games matching the alias
+    let aliasGames: Game[] = [];
+
+    for (alias of aliases) {
+      aliasGames = aliasGames.concat(Game.getGamesByAlias(alias));
```

Comment:
What happens if I provide two aliases for the same game?

Reply:
Duplicates now get removed.